index.html

```diff
@@ -121,6 +121,26 @@ <h3 class="project-title"><a href="https://github.com/kunjgit/GameZone/tree/main
           </a>
         </li>
 
+        <li class="project-item  active" data-filter-item data-category="open source">
+          <a href="./Games/Trivia_It/">
+
+            <figure class="project-img">
+              <div class="project-item-icon-box">
+                <img
+                  src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Activities/Video%20Game.png"
+                  alt="Eye" width="3" />
+              </div>
+
+              <img src="./assets/images/Trivia_It.png" alt="trivial quiz game" loading="lazy">
+            </figure>
+
+            <h3 class="project-title"><a href="https://github.com/kunjgit/GameZone/tree/main/Games/Trivia_It" target="_blank">4. Trivia It! 🔗</a></h3>
+
+            <p class="project-category">Test your RomCom & movie knowledge.</p>
+
+          </a>
+        </li>
+
```

Comment:
Make sure you don't make changes in main index file as we are adding games it will create conflicts !! 😄 
Also add your entries in main README file !!
other things are done 🔥

Reply:
I have removed the above lines of code from index.html
And I have attached an image of the changes I understood you wanted me to make.

<img width="960" alt="preview" src="https://github.com/kunjgit/GameZone/assets/112249538/7f47031a-9efe-410e-b0cf-0b14d59e731d">

If I am right about all of them, can I submit the changes ?

Reply:
Hey @kunjgit 
I have removed the code I added to index.html and updated the main README.md

<img width="674" alt="image" src="https://github.com/kunjgit/GameZone/assets/112249538/4ee203f2-b290-4314-96b3-9fa112dc5d69">

I hope this resolves the issue. Thank you :)